CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Change st2packs definition to a list, to support multiple st2packs containers (#166) (by @moonrail)
 * Enabled RBAC/LDAP configuration for OSS version, removed enterprise flags (#182) (by @hnanchahal)
 * Fixed datastore_crypto_key secret name for rules engine (#188) (by @lordpengwin)
+* Updated redis constant sentinel ID which will allow other sentinel peers to update to the new given IP in case of pod failure or worker node reboots. (#191) (by @manisha-tanwar)
```

Comment:
Added CHANGELOG too @armab 
Can you please review.

Reply:
Please move the Changelog record under the `In Development` section, otherwise looks good